File: ejercicios-js/ejercicio2.js
// Juego de adivinar el número
let intento;
const secreto = Math.floor(Math.random() * 100) + 1; // Generar el número secreto solo una vez

do {
  intento = Math.floor(Math.random() * 100) + 1; // Simula un intento aleatorio
  if (intento < secreto) {
    console.log(`${intento} es menor ❌`);
  } else if (intento > secreto) {
    console.log(`${intento} es mayor ❌`);
  }
} while (intento !== secreto);

console.log(`¡Correcto! El número era ${secreto} ✅`);

File: ejercicios-py/ejercicio2.py
import random

# Juego de adivinar el número
secreto = random.randint(1, 100)
intento = None

while intento != secreto:
    intento = random.randint(1, 100)  # Simula un intento aleatorio
    if intento < secreto:
        print(f"{intento} es menor")
    elif intento > secreto:
        print(f"{intento} es mayor")

print(f"¡Correcto! El número era {secreto}")